proposed/0002-release-schedule.md

```diff
@@ -0,0 +1,118 @@
+# Release Schedule
+
+## Summary
+
+Defining the Schedule and rules for new releases
+
+## Motivation
+
+Its important to have a reliability for long term plans, we can offer this by following a consistent schedule with hard rules.
+Different People/Users have different needs. This RFC allows to find a middleground, which satisfies all of our users, but also will show us, what our users actually need and want, providing a base for a decision which will be seen as good enough to not need a bigger change for at least a few years.
+With this RFC we then also have something to reference in case of future arguments about Releases and what gets into them.
+
+## Detailed Explanation
+
+With this RFC we will define a basic scheme with rules by which we will do releases, have and create release branches, and also by which we decide what kind of changes get when merged into which release branch.
+
+## Rationale and Alternatives
+
+
+{{Discuss 2-3 different alternative solutions that were considered. This is required, even if it seems like a stretch. Then explain why this is the best choice out of available ones.}}
+
+### Alternative 1
+
+Versioning follows semver.org with these tailored definitions:
+
+- MAJOR version when you make incompatible API changes
+  - public methods with changed signatures (e.g. new required parameter)
+  - removed code that may still be used by some (e.g. unused lib/Zend code or IE8 compat)
+  - rename things that could affect users (e.g. xml namespace)
+  - dependencies updated in major ways (e.g. dropping support of PHP 7.4 or MySQL 5.6)
+- MINOR version when you add functionality in a backwards compatible manner
+  - changed code that is very unlikely to have any effect on large majority of users although not guaranteed
+  - typical usage is not expected to be affected, only if user was not following best practices
+  - no regressions foreseen, but there is always a risk a new feature affects some in unexpected ways
+  - performance/stability is improved significantly with little or no negative impacts expected
+  - dependencies updated to fix bugs or bring in minor improvements
+- PATCH version when you make backwards compatible bug fixes
+  - best case: fix has zero negative impact
+  - worst case: security fix is urgent enough to justify potentially breaking user's stores in a small way (e.g. improving malicious string filtering)
+
+Given the above, the rules for creating PRs, merging, branching and tagging releases shall be as follows:
+
+- Release timing is flexible (determined by volunteer time availability/motivation/activity) but ideally:
+  - roughly 6-12 MINOR releases per year
+  - no more than two MAJOR releases per year (since each major release carries some overhead for PATCH updates)
```

Comment:
if there are 2 major releases per years and we have to support each branch for 2 years it mean we'll have 4 active branches + main and next?